src/convertToObject.js

```diff
@@ -3,16 +3,24 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties (
+ * see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesCSS = {};
+  const arr = sourceString.split(';').filter(item =>
+    item.trim()).map(item => item.split(':'));
+
+  arr.forEach(item => (stylesCSS[item[0].trim()] = item[1].trim()));
```

Comment:
Add changes according to 2 and 4 points from [checklist](https://github.com/mate-academy/js_style-to-object/blob/master/checklist.md)